stable/xray/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [1.1.2] - Oct 3, 2019
```

Comment:
@kobisony1 I think you should bump the chart version to 1.2.0 since we had a minor bump in Xray

Reply:
+1